src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
@jturcotte DontAlign gives me lines like
```
    QObject::connect(
        _requestEtagJob, SIGNAL(etagRetreived(QString)), this, SLOT(etagRetreived(QString)));
    connect(
        _engine.data(), SIGNAL(finished(bool)), SLOT(slotSyncFinished(bool)), Qt::QueuedConnection);
```
I guess clang-format realizes this have to be two lines and then prefers not splitting the argument list. And yet, AlwaysBreak has a bunch of not-pretty cases. I could live with the current setting or with DontAlign -- @ogoffart, do you have opinions?